x-pack/plugins/lens/public/visualizations/datatable/utils.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ColumnState } from '../../../common/expressions';
+
+export function getColumnAlignment(
+  alignment: ColumnState['alignment'],
+  isNumeric = false
+): 'left' | 'right' | 'center' {
+  if (alignment) return alignment;
+  return (isNumeric ? 'right' : 'left') ?? 'right';
```

Comment:
nit: I do think the `?? 'right'` is not required here as the ternary condition will always return a non-`undefined` value.

Reply:
Removed in 0ab6417